Frameworks/UIKit/StarboardXaml/ApplicationCompositor.h

```diff
@@ -25,7 +25,15 @@ extern "C" void RunApplicationMain(Platform::String^ principalClassName,
                                    float windowWidth,
                                    float windowHeight,
                                    ActivationType type,
-                                   Platform::String^ activationArg);
+                                   Platform::Object^ activationArg);
+
+// Because String^ does not inherit from IInspectable
```

Comment:
nit: please make comments full English sentences. A fragment like this almost always has an implied context that a casual observer may not pick up on. In this case there is an implied "Why are there two of the almost identical functions here" but someone unfamiliar with C++\Cx at all may be internally asking very different questions about what is going on here.